src/main/scala/org/virtuslab/internal/load/reader/ReaderState.scala

```diff
@@ -2,9 +2,10 @@ package org.virtuslab.internal.load.reader
 
 sealed trait ReaderState:
   def indent: Int
-
 case object ReaderState:
-  case class Stream(indent: Int)   extends ReaderState
-  case class Document(indent: Int) extends ReaderState
+  case object Stream extends ReaderState:
+    override val indent = 0
```

Comment:
If you don't use val indent in Stream and Document, maybe you should delete them.

Reply:
Yeah, you're right.